Description: Fix fails to build due to missing files in POTFILES.in
Author: David Michael Smith <sidicas2@gmail.com>
Forwarded: Yes
Bug: https://sourceforge.net/p/liferea/bugs/1115/ 
Last-Update: 2013-09-17
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -180,6 +180,9 @@
 src/notification/notification.c
 src/notification/notification.h
 src/webkit/webkit.c
+src/fl_sources/inoreader_source.c
+src/fl_sources/reedah_source.c
+src/fl_sources/reedah_source_feed.c
 [type: gettext/glade]glade/auth.ui
 [type: gettext/glade]glade/google_source.ui
 [type: gettext/glade]glade/inoreader_source.ui
